strings/accent_universal.json

```
@@ -92,6 +92,9 @@
 		"cringe": "swive",
 		"bussing": "splendid",
 		"bussin": "splendid",
+		"lmao": "ha ha",
+		"lol": "heh",
+		"femboy": "girly boy",
```

```suggestion
		"femboy": "girly boy",
		"wtf": "what the devil",
```
Small suggestion for you. But wtf could be added to this list too.